EssentialsDiscord/src/main/java/net/essentialsx/discord/DiscordSettings.java

```diff
@@ -107,6 +108,10 @@ public boolean isAlwaysReceivePrimary() {
         return config.getBoolean("always-receive-primary", false);
     }
 
+    public boolean isUseEssentialsEvents() {
+        return config.getBoolean("use-essentials-events", false);
```

Comment:
Default to true here, users won't expect to have local chat relayed by default.

```suggestion
        return config.getBoolean("use-essentials-events", true);
```